src/app/clusters/descriptor/descriptor.cpp

```diff
@@ -89,6 +89,7 @@ class DescriptorAttrAccess : public AttributeAccessInterface
     CHIP_ERROR ReadServerClusters(EndpointId endpoint, AttributeValueEncoder & aEncoder);
     CHIP_ERROR ReadClusterRevision(EndpointId endpoint, AttributeValueEncoder & aEncoder);
     CHIP_ERROR ReadFeatureMap(EndpointId endpoint, AttributeValueEncoder & aEncoder);
+    CHIP_ERROR ReadEndpointUniqueId(EndpointId endpoint, AttributeValueEncoder & aEncoder);
```

Comment:
It's odd to have the declaration unconditional but the implementation conditional.